miniml/compiler/compile.scm

```diff
@@ -446,6 +450,23 @@
     (cons "with" (cons 'WITH #f))
     ))
 
+(define operator-kw (list
+   (cons "&&" (cons 'AMPERAMPER #f))
+   (cons "@@" (cons 'ATAT #f))
+   (cons "!=" (cons 'INFIXOP0 "!="))
+   (cons "|" (cons 'BAR #f))
+   (cons "||" (cons 'BARBAR #f))
+   (cons "|>" (cons 'BARGT #f))
+   (cons "=" (cons 'EQ #f))
+   (cons "<-" (cons 'LTMINUS #f))
+   (cons "-" (cons 'MINUS #f))
+   (cons "->" (cons 'MINUSGT #f))
+   (cons "?" (cons 'QUESTION #f))
+   (cons "*" (cons 'STAR #f))
+   (cons "~" (cons 'TILDE #f))
+   ))
```

Comment:
This would read nicer quoted:

```scheme
(define operator-kw '(
   ("&&" AMPERAMPER #f)
   ("@@" ATAT #f)
   ...)
```

Reply:
That would actually need to be `("&&" AMPERAMPER . #f)`, which I find less readable as it isn't clear that `"&&"` is the key and `('AMPERAMPER . #f)` the value.

Reply:
Ah, indeed.